WalletWasabi/Crypto/Randomness/Characters.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.Crypto.Randomness
+{
+	public class Characters
```

Comment:
Not sure you need to introduce this class as it increases complexity.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/3903/commits/2e60f722f8eca5cce325a57e1baf41a7926fea17